internal/handler/msg_hello.go

```diff
@@ -73,9 +73,15 @@ func (h *Handler) MsgHello(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 		)
 	}
 
-	mechs, err := h.getUserSupportedMechs(ctx, db, username)
-	if err != nil {
-		return nil, lazyerrors.Error(err)
+	// for the backend authentication return `PLAIN` regardless of the user existence,
+	// getting backend users is not easy
```

Comment:
Well, it is not about being "easy" – those backend users may not exist (SQLite) or be unrelated users (PostgreSQL)

Reply:
Sure, let's remove that comment :+1: